tests/test_utils.py

```diff
@@ -140,6 +141,7 @@ def evaluate_json_test(dataset, expectation_type, test):
     """
 
     dataset.set_default_expectation_argument('result_format', 'COMPLETE')
+    dataset.set_default_expectation_argument('catch_exceptions', True)
```

Comment:
To see the impact of this change, I added the following to evaluate_json_test:
```
        if not ('success' in test['out'] or 'traceback_substring' in test['out']):
            warnings.warn("This test does not test success or traceback_substring. Is it defined improperly?")

        if result['exception_info']['raised_exception'] is True and 'traceback_subsring' not in test['out']:
            warnings.warn("This test did not test a traceback substring, but the expectation raised an exception.")
```

That did trigger extra warnings, so I think we need to double check the tests throughout. Probably not an issue with your PR, but I want to look into it while you make the other changes.